auth-web/src/components/auth/manage-business/AddNameRequestForm.vue

```diff
@@ -181,8 +182,10 @@ export default class AddNameRequestForm extends Vue {
     }
   }
 
-  private formatNrNumber (): void {
+  protected async formatNrNumber (): Promise<void> {
     this.nrNumber = CommonUtils.formatIncorporationNumber(this.nrNumber, true)
+    await Vue.nextTick() // wait for component to update
+    this.$refs.nrNumber.validate()
```

Comment:
When the user leaves the NR input component...
1. format the NR
2. wait for component to update
3. apply validation using rules

See ticket for test cases.